.devcontainer/docker-compose-local.yaml

```diff
@@ -70,15 +70,15 @@ services:
       - "localhost:host-gateway"
 
   backend:
-    build:
-      context: ..
-      dockerfile: ".devcontainer/Dockerfile.backend"
-    platform: linux/amd64
+    # TODO: Change back to build
+    image: lumiuv:1
+    command: ["uv", "run", "uvicorn", "--host", "0.0.0.0", "app.main:app"]
```

Comment:
do you have thoughts on whether it should be `app.main` or `lumigator.main`? The latter feels easier to read in that you know specifically which app/library you're working with.